arbnode/batch_poster.go

```diff
@@ -622,6 +629,23 @@ func (b *BatchPoster) Start(ctxIn context.Context) {
 	b.redisLock.Start(ctxIn)
 	b.StopWaiter.Start(ctxIn, b)
 	b.CallIteratively(func(ctx context.Context) time.Duration {
+		var err error
+		if common.HexToAddress(b.config().GasRefunderAddress) != (common.Address{}) {
+			var gasRefunderBalance *big.Int
+			gasRefunderBalance, err = b.l1Reader.Client().BalanceAt(ctx, common.HexToAddress(b.config().GasRefunderAddress), nil)
+			if err != nil {
+				log.Warn("error fetching batch poster gas refunder balance", "err", err)
+			}
+			batchPosterGasRefunderBalance.Update(float64(gasRefunderBalance.Int64()) / params.Ether)
```

Comment:
I think if an error occurs, `gasRefunderBalance` might be nil which would trigger a panic, so we should put this line of code in an else statement. This applies to the other balance checks as well.

Reply:
done.